app/models/purl.rb

```diff
@@ -1,8 +1,10 @@
-class Purl < ApplicationRecord # rubocop:disable Metrics/ClassLength
+class Purl < ApplicationRecord
   has_and_belongs_to_many :collections
   has_many :constituent_memberships, inverse_of: :parent, dependent: :destroy
   has_many :constituents, -> { order 'constituent_memberships.sort_order' },
            through: :constituent_memberships, source: :child
+  has_many :parent_memberships, class_name: 'ConstituentMembership', inverse_of: :child, dependent: :destroy
```

Comment:
Is this unrelated?

Reply:
No, this is how we get the constituents for virtual objects

Reply:
Gotcha.